open_schools_platform/parent_management/families/views.py

```diff
@@ -84,4 +85,6 @@ def post(self, request):
             raise NotAcceptable("Parent is already in this family")
         query = create_query(sender_model_name="family", sender_id=family.id,
                              recipient_model_name="parentprofile", recipient_id=parent.id)
+        notify_user(user=parent.user, title='Вы были приглашены в семью!',
+                    body=f'{family.name} пригласила вас к себе!', data={"query": str(query.id)})
```

Comment:
it would better to place this message in special file like it was made in OrganizationConstants